middlewares/osio/auth.go

```diff
@@ -17,6 +17,15 @@ const (
 	UserIDParam   = "identity_id"
 )
 
+type RequestType string
+
+const (
+	api     RequestType = "api"
+	metrics RequestType = "metrics"
+	console RequestType = "console"
+	logs    RequestType = "logs"
+)
+
```

Comment:
RequestType is introduce to categorize request and different methods are added to provide type specific details.